<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sales_add_order</class>
 <widget class="QWidget" name="sales_add_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>811</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="order_index">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_num">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>订单号</string>
   </property>
  </widget>
  <widget class="QLabel" name="products_num">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>产品号</string>
   </property>
  </widget>
  <widget class="QLabel" name="sales_spot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>销售点</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_num">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>顾客id</string>
   </property>
  </widget>
  <widget class="QLabel" name="pay_mode">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>支付方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_price">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>总价</string>
   </property>
  </widget>
  <widget class="QLabel" name="place_ordertime">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>下单时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="order_num_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="products_num_edit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_num_edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_price_edit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pay_mode_edit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sales_spot_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="place_ordertime_edit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_order_pushb">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_order_pushb">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_order_pushb">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_order_pushb">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
